### DISCUSSION TITLE
Add option for inner gaps only

### DISCUSSION THREAD

**Discussion by wesbluemarine** [YaLTeR/niri#81] [Ideas]
I would like to have gaps only between windows.

**Comment by YaLTeR** [comment]
Not even gaps to the left of the leftmost window and to the right of the rightmost?

**Reply by wesbluemarine** [reply]
To the left of the leftmost window and to the right of the rightmost it's OK to have gaps for me, but it could be better to have settings for it, like other compositor/window managers.

**Reply by salman-farooq-sh** [reply]
> Not even gaps to the left of the leftmost window and to the right of the rightmost?

Outer gaps just make the display bezels feel bigger than they really are when you have a wallpaper which blends with the display bezels e.g. pitch black wallpaper on glass-covered amoled laptop display.

**Comment by megatux** [comment]
I also want this. Gaps only between windows makes sense, to differentiate them but not on the monitor edges (or panels), it's a waste of space. I think i3 has inner and outer gaps for this. See [this](https://www.reddit.com/r/i3wm/comments/ne9smm/how_do_i_get_gaps_between_the_windows_but_not_to/)
